gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/services/llm/llm_service.py

```diff
@@ -30,19 +32,25 @@
 logger = logging.getLogger(__name__)
 
 
-async def check_llm_setting(setting: LLMSetting) -> bool:
+async def check_llm_setting(query: LLMProviderSettingStatusQuery) -> bool:
     """
     Run a check for a given LLM setting.
 
     Args:
-        setting: The Large Language Model setting to check
+        query: The query for the LLM Provider Setting Status
 
     Returns:
          True for a valid LLM setting. Raise exception otherwise.
     """
 
     logger.info('Get the LLM Factory, then check the LLM setting.')
-    return await get_llm_factory(setting).check_llm_setting()
+    langfuse_callback_handler = None
```

Comment:
```suggestion
    observability_callback_handler = None
```

And refactor the next lines.